capstone-frontend/src/pages/BorderPage.jsx

```diff
@@ -1,101 +1,105 @@
+    borderNum: "0",
     userInput: "",
   });
 
-  useEffect(() => {
-    fetchComments();
-    fetch("http://localhost:5000/borderdata")
-      .then((response) => response.json())
-      .then((data) => {
-        setCrossingData(data);
-      });
-  }, []);
+  const { data, error, isLoading } = useSWR(
+    "http://localhost:5000/borderdata",
+    fetcher
+  );
+  const {
+    data: data2,
+    error: error2,
+    isLoading: isLoading2,
+  } = useSWR("http://localhost:5000/usersposts", fetcher);
 
-  function handle(e) {
-    const newdata = { ...postData };
-    newdata[e.target.id] = e.target.value;
-    setPostData(newdata);
-  }
-  function submit(e) {
-    e.preventDefault();
-    fetch("http://localhost:5000/usersposts", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-        "Access-Control-Allow-Origin": "http://localhost:5000/usersposts",
-      },
-      body: JSON.stringify(postData),
-    })
-      .then((response) => response.json())
-      .then((data) => console.log(data))
-      .catch((error) => console.error(error));
-  }
+  const crossingData = data;
+  const allComments = data2;
+
+  useEffect(() => {
+    if (allComments) {
+      createCommentDisplays(allComments);
+    }
+  }, [allComments]);
 
-  function fetchComments() {
-    fetch("http://localhost:5000/usersposts")
-      .then((response) => response.json())
-      .then((data) => {
-        specificBorderPosts = [];
-        for (const item of data) {
-          if (data[item].borderNum == borderIndex) {
-            specificBorderPosts.push(data[item]);
-          }
-        }
-      });
+  let allRelevantComments = [];
+  let populatedDivs = [];
+  function createCommentDisplays(specificBorderPosts) {
+    allRelevantComments = [];
+    specificBorderPosts.forEach((element) => {
+      if (parseInt(element.borderNum) == borderIndex) {
+        allRelevantComments.push(element.userInput);
+      }
+    });
+    populatedDivs = allRelevantComments.map((comment) => <div>{comment}</div>);
   }
 
-  if (crossingData.length > 0) {
+  if (crossingData) {
+    const thisBorder = crossingData.allMexicanPorts[borderIndex];
     return (
       <div>
         <Header />
-        <div>yay border </div>
         <div className="border">
-          <div>{thisborder.border[0]}</div>
-          <div>{thisborder.borderRegion[0]}</div>
-          <div>{thisborder.crossingName[0]}</div>
-          <div>{thisborder.hours[0]}</div>
-          <div>{thisborder.passengerVehicleWait[0]}</div>
-          <div>{thisborder.pedestrianLaneWait[0]}</div>
-          <div>{thisborder.portStatus[0]}</div>
+          <div>{thisBorder.border}</div>
+          <div>{thisBorder.borderRegion[0]}</div>
+          <div>{thisBorder.crossingName[0]}</div>
+          <div>{thisBorder.hours[0]}</div>
+          <div>{thisBorder.passengerVehicleWait[0]}</div>
+          <div>{thisBorder.pedestrianLaneWait[0]}</div>
+          <div>{thisBorder.portStatus[0]}</div>
```

Comment:
nit: not sure `div` would be the best element choice for this piece (depending what you're trying to convey here).

Seems like you're switching to a component library of some kind so I'd suggest using appropriate components from there instead